README.md

```diff
@@ -43,6 +43,28 @@ The documentation contains information about how to setup and run the two planne
 
 ## Installation
 
+### Script Installation (Recommended)
+1. Go to the [PX4 Development Guide](https://dev.px4.io/v1.8.2/en/setup/dev_env_linux.html#gazebo-with-ros) and run the Gazebo with ROS installation script
```

Comment:
This is a specific version of the setup for PX4 1.8. Not sure if that is intentional. Master is at http://dev.px4.io/master/en/setup/dev_env_linux_ubuntu.html#rosgazebo

Reply:
Also, did you test these? I've been running into problems on the other scripts due to recent CMAKE/Python issues
ie nothing wrong with your steps, just that the scripts might break.

Reply:
Yes, it was intentional. That's the last script that installs ROS Kinetic instead of Melodic. I didn't test the entire thing yet

Reply:
Sounds good! Up to you if you want to spell that out in a note (i.e. so that people don't just use master version without thinking about it.